### PR TITLE
Add formula support on cell writing

### PR DESCRIPTION
**Pull request by rbaraffe**

### CODE REVIEW

File: services/cell_test.go [hunk 1]
```diff
@@ -37,7 +37,12 @@ func TestWriteCell(t *testing.T) {
 		}),
 	}
 
-	err := file.writeCell("test", "myTestValue", styles, "A2")
+	cellString := &models.Cell{
```
Comment (Jerome1337) [inline services/cell_test.go]: cell alone, cellString is a weird name

File: services/cell_test.go [hunk 2]
```diff
@@ -47,4 +52,17 @@ func TestWriteCell(t *testing.T) {
 	styleIndex, err := file.Excel.GetCellStyle("test", "A2")
 	assert.Nil(t, err)
 	assert.Equal(t, styleIndex, 1, "Style index should be \"1\"")
+
+	cellFormula := &models.Cell{
```
Comment (Jerome1337) [inline services/cell_test.go]: cellWithFormula instead?

File: services/cell_test.go [hunk 3]
```diff
@@ -47,4 +52,17 @@ func TestWriteCell(t *testing.T) {
...
+		Style:     styles,
+		IsFormula: true,
+	}
+
+	err = file.writeCell("test", cellFormula, "A5")
+	assert.Nil(t, err)
+
+	value, err = file.Excel.GetCellFormula("test", "A5")
+	assert.Nil(t, err)
+	assert.Equal(t, value, "SUM(B2:B5)", "Value should be \"SUM(B2:B5)\"")
```
Comment (Jerome1337) [inline services/cell_test.go:67]: This getCellValue should return the expected formula value or the formula?

File: roadmap.md [hunk 4]
```diff
@@ -5,26 +5,28 @@ Excelante is an Excel API that allow user to create or to read some excel file.
...
-    - ~~List of style must be present in readme~~
-  - ~~You can read an excel file and extract columns values~~
-   - ~~This service take an excel file and a list of fileds (or * ) to extract in input.~~
-   - ~~Json file with the values is returned by the service.~~
-  - ~~Optimisation & best practices~~
-   - ~~Using all the best practices to have a good & stable starter solution, that can be used in production environment~~
-   - ~~Set up deployment options~~
-     - ~~Docker ready~~
-  - ~~Create excel graph with json input file~~
+
```
Comment (Jerome1337) [inline roadmap.md]: Your branch seems outdated, maybe try to update it from master

**Review by Jerome1337 — CHANGES_REQUESTED** [review]

File: services/cell_test.go (hunk 3 above)
Comment (rbaraffe) [reply services/cell_test.go:67]: GetCellFormula should return the formula, not the calculated value

**Review by Jerome1337 — APPROVED** [review]

**Pull request merged**